### ISSUE TITLE
Says incorrect API key provided even after switching

### ISSUE THREAD

**Issue by romanpTAMU** [WolframResearch/Chatbook#489]
Changed my API key and put in my new one when asked for an API key, yet it still uses my old API key and throws an error, and won't pop up the authentication window.

<details>
<summary>Debug Data</summary>

| Property | Value |
| --- | --- |
| Name | ``"Wolfram/Chatbook"`` |
| Version | ``"1.0.0"`` |
| EvaluationEnvironment | ``"Session"`` |
| FrontEndVersion | ``"Desktop: 13.3 for Microsoft Windows (64-bit) (June 3, 2023)"`` |
| KernelVersion | ``"13.3.0 for Microsoft Windows (64-bit) (June 3, 2023)"`` |
| SystemID | ``"Windows-x86-64"`` |
| Notebooks | ``True`` |
| DynamicEvaluation | ``False`` |
| SynchronousEvaluation | ``False`` |
| TaskEvaluation | ``False`` |

## Settings

```
Inherited
```

## Stack Data
```
throwFailure
trimURL
```

## Failure Data

```
<|
  "Evaluation" :> Wolfram`Chatbook`Actions`Private`writeChunk[Dynamic[Wolfram`Chatbook`Actions`Private`container$56114], CellObject["584f0699-2334-4c42-b12c-b65d8704bea1", "c1874b0e-a856-4026-b4c2-81a138ff8155"], "W6xKpbfxzpgWp3i4ctUpJj\",\"object\":\"chat.completion.chunk\",\"created\":1702229641,\"model\":\"gpt-3.5-turbo-0613\",\"system_fingerprint\":null,\"choices\":[{\"index\":0,\"delta\":{\"role\":\"assistant\",\"content\":\"\"},\"finish_reason\":null}]}\n\n"]
  "Arguments" -> {}
|>
```

</details>

**Comment by raverydavis** [comment]
I noticed you're on an older first, first please update to a recent version using `PacletInstall["Wolfram/Chatbook", UpdatePacletSites -> True]`

Trigger the service connection to prompt for the key with `ServiceConnect["OpenAI", "New"]` Checkmark save connection if you want to keep using it for future evaluations

Afterwards you can manage the connections by viewing the saved connections with ```ServiceConnections`SavedConnections[]```
If you want to delete a saved connection ```ServiceConnections`DeleteConnection[ServiceObject]``` where `ServiceObject` is one of the objects printed by `SavedConnections[]`

**Comment by rhennigan** [comment]
This error was fixed in #379
